config.md

```diff
@@ -265,6 +265,27 @@ Annotations are key-value maps.
 }
 ```
 
+## HypervisorPath
+
+HypervisorPath is an optional absolute path to the hypervisor binary used by VM-based runtimes.
+
+## ImagePath
+
+ImagePath is an optional absolute path to the root filesystem image used by VM-based runtimes.
```

Comment:
When `ImagePath` and `root.path` are both specific, the rootfs  will the the
`filepath.Join(ImageMountPoint, root.path)`  (root.path can be absolute or not)
the `ImageMountPoint` is mounted by the guest inside the VM. the `ImagePath` is the block device passed to VM by the hypervisor.

Take docker's devicemapper dirver as an example, `ImagePath` will  be the path of the block device of the dm snapshot, and `root.path` will be "rootfs" or "/rootfs" or "./rootfs". (docker dons't put the real rootfs in the root directory of the block device. it put the real rootfs in /rootfs inside the block device).